analyzers/rspec/vbnet/Sonar_way_profile.json

```diff
@@ -97,6 +97,7 @@
     "S3926",
     "S3927",
     "S3949",
+    "S3966",
```

Comment:
This was removed from SonarWay in C#, should be the same for VB

Reply:
@pavel-mikula-sonarsource  when?
On master this is sonar-way https://github.com/SonarSource/rspec/blob/master/rules/S3966/csharp/metadata.json#L26

Reply:
Ah, ok. My bad